### ISSUE TITLE
Experiment setting, experiment results

### ISSUE THREAD

**Issue by weidandan1997** [VIS-VAR/LGSC-for-FAS#28]
Hello,author.
I have several problems to ask you for help.
First,I want to know what you mean in the paper " We resample the training examples to keep the live-spoof ratio to 1:1. ".Do you mean the number of  live and spoof images in each batch is 1:1?  Or  is  1:1   only for training set ,not for each batch?
Second,I want to know how many images do you resample for one video.
Third,  I  have implemented some experiments, but the result is not good.  The result in the protocol1,2 of OULU  is not good.The results in bad in the protocol3 of SIW. The results is worse in the Replay-to-Casia.

Here are some spoof  cues. I dont know why the spoof cues are so different in diffent dataset. And I dont know why there are always some Highlights in each spoof cues. 
oulu_protocol1
![image](https://user-images.githubusercontent.com/52347040/95828580-bbe3e780-0d67-11eb-8bee-2333d03c03d0.png)
![image](https://user-images.githubusercontent.com/52347040/95828637-cef6b780-0d67-11eb-8ba5-ea0737c1c86e.png)
 oulu_protocol2
![image](https://user-images.githubusercontent.com/52347040/95828828-06656400-0d68-11eb-984c-bd246da0c154.png)
![image](https://user-images.githubusercontent.com/52347040/95828854-0ebd9f00-0d68-11eb-9393-1b97285d09c8.png)

siw_protocol2
![image](https://user-images.githubusercontent.com/52347040/95828908-239a3280-0d68-11eb-9ed9-e8766ca5e94c.png)
![image](https://user-images.githubusercontent.com/52347040/95828949-314fb800-0d68-11eb-9c10-c9a3d1ef1038.png)

Help!

**Comment by adamhtoolwin** [comment]
It seems your images are in a different color format to RGB. Or are they normalized?

**Comment by weidandan1997** [comment]
Yes,the images are in a BGR color format.

**Comment by weidandan1997** [comment]
> It seems your images are in a different color format to RGB. Or are they normalized?

Yes,the images are in a BGR color format.

**Comment by adamhtoolwin** [comment]
The input to the model has to be RGB if I am not mistaken. Or is this only for visual purposes?

Just to share some of my experiences, I got an ACER of around 0.1 on the SiW-M dataset and similar on protocol 4 of the OULU dataset. However, I am not using paddle but the pytorch version (heavily modified version of this [repo](https://github.com/Podidiving/lgsc-for-fas-pytorch) credit to [Poddiving](https://github.com/Podidiving)). But the performance drops when tested on a new dataset (or even different environment actually).

**Comment by CHNxindong** [comment]
> The input to the model has to be RGB if I am not mistaken. Or is this only for visual purposes?
> 
> Just to share some of my experiences, I got an ACER of around 0.1 on the SiW-M dataset and similar on protocol 4 of the OULU dataset. However, I am not using paddle but the pytorch version (heavily modified version of this [repo](https://github.com/Podidiving/lgsc-for-fas-pytorch) credit to [Poddiving](https://github.com/Podidiving)). But the performance drops when tested on a new dataset (or even different environment actually).

@AdamHtooLwin hello, I am also training on Oulu on the pytorch version code, But I can not get a good results as you, I get ACER 1.0 on Protocol1 and ACER 3.3 on Protocol2, can you share how to train it on this code, do you modify some code? Or do you have some good advice for me? Thank you! Thank you! Thank you!!!

**Comment by adamhtoolwin** [comment]
I suggest you take a look at this [repo](https://github.com/Podidiving/lgsc-for-fas-pytorch). I based my code off of this and it's Pytorch based too. And maybe give him a star too :)

**Comment by PangziZhang523** [comment]
Hello, have you solved this Highlights problem in each spoof cues?

**Comment by weidandan1997** [comment]
> Hello, have you solved this Highlights problem in each spoof cues?
Sorry,I haven't been following this problem.

**Comment by Aluooooo** [comment]
Have you solved this problem?
